docs/en/user_guides/1_config.md

```diff
@@ -255,6 +287,26 @@ model = dict(
     auxiliary_head=dict(...))
 ```
 
+```python
+from mmengine.config import Config
+
+cfg = Config.fromfile('configs/pspnet/pspnet_r50-d8_4xb2-40k_cityscapes-512x1024.py')
+print(cfg.model.backbone)
```

Comment:
I have little question why there is load `configs/pspnet/pspnet_r50-d8_4xb2-40k_cityscapes-512x1024.py` file, as both of above code block and the description have nothing to do with this file.

My suggestion is complete this example. Like tell users if you would like to modify something (the more specific the better), you can write a new config file and import the base config and modify the value, and here is an example